tests/Extensions/Hosting.Services.Web.UnitTests/TestHelpers/SfConfigurationProviderHelper.cs

```diff
@@ -2,15 +2,20 @@
 // Licensed under the MIT license.
 
 using System;
-
+using Microsoft.Omex.Extensions.ServiceFabricGuest.Abstractions;
+
 namespace Microsoft.Omex.Extensions.Hosting.Services.Web.UnitTests
 {
 	internal static class SfConfigurationProviderHelper
-	{
+	{
+		/// SF environment variables taken from https://docs.microsoft.com/en-us/azure/service-fabric/service-fabric-environment-variables-reference

```

Comment:
This comment should be in `SfConfigurationProvider`. And here we should point to that class instead since values should be instead (alternatively we can access internal fields from of SfConfigurationProvider here by moving class to Abstractions UnitTest and then referencing it).

Reply:
Done